.github/actions/aws-cni/k8s-versions.yaml

```diff
@@ -5,13 +5,11 @@ include:
     region: us-west-1
   - version: "1.27"
     region: us-east-2
-    ipsec: true
```

Comment:
Is there any concerns with reducing the ipsec coverage on AWS CNI tests and just relying on the EKS one?

Reply:
Good concern. There is an ongoing discussion about IPsec configs for AWS-CNI / EKS. We will follow up in a separate PR.